### PR TITLE
borges chart

### PR DESCRIPTION
**Pull request by rporres**

### CODE REVIEW

File: borges/templates/consumer-deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+            {{ end }}
+
+          ports:
+            {{ if .Values.consumer.profiler }}
+            - containerPort: {{ .Values.consumer.profilerPort }}
+            {{ end }}
+            {{ if .Values.consumer.metrics }}
+            - containerPort: {{ .Values.consumer.metricsPort }}
+            {{ end }}
+          env:
```
Comment (ajnavarro) [inline borges/templates/consumer-deployment.yaml]: Just to have in mind, we are going to need a new environment variable in the near future to bucketize the borges output: https://github.com/src-d/borges/pull/184/files#diff-04c6e90faac2675aa89e2176d2eec7d8R59

Comment (rporres) [reply borges/templates/consumer-deployment.yaml]: What consequences does it have in how the program is started?

Comment (rporres) [reply borges/templates/consumer-deployment.yaml]: Ok, I guess we can just add it when the time comes. Feel free to create a PR 😍

Comment (ajnavarro) [reply borges/templates/consumer-deployment.yaml]: perfect! we'll do that!

**Pull request merged**